packages/web/src/components/edit/fields/visibility/VisibilityField.tsx

```diff
@@ -197,16 +200,18 @@ const VisibilityMenuFields = (props: VisibilityMenuFieldsProps) => {
   const { isEnabled: isPaidScheduledEnabled } = useFeatureFlag(
     FeatureFlags.PAID_SCHEDULED
   )
-  const { initiallyPublic, entityType } = props
+  const { initiallyPublic, isAllowedToPublish, entityType = true } = props
   const [field] = useField<VisibilityType>('visibilityType')
 
   return (
     <RadioGroup {...field}>
-      <ModalRadioItem
-        value='public'
-        label={messages.public}
-        description={messages.publicDescription}
-      />
+      {isAllowedToPublish ? (
```

Comment:
I think just removing the option might not be what we want?
In these cases we usually disable the option and show a tooltip explaining the reason it's disabled (e.g. collectible gated when you don't have any collectibles)

Reply:
good call^ will update

Reply:
ty! You can probably take the tooltip copy from `PublishButton` which has a disabled state with tooltip

Reply:
nice! 
![image](https://github.com/user-attachments/assets/1ba1df9a-c426-448a-b217-c85b9df52970)
updated :)